frontend/public/components/utils/storage-class-dropdown.tsx

```diff
@@ -175,6 +180,7 @@ const StorageClassDropdownNoStorageClassOption = props => {
   </div>;
 };
 
+
 export type StorageClassDropdownInnerState = {
   items: any;
```

Comment:
This would be a follow on issue since it's not new, but this type should be improved. I think it's okay to use any as a placeholder, but there should probably be a FIXME/TODO comment here to come back around and improve the specificity of this type.

Reply:
Actually, see my review comment. Since we need a return type for the `getUnorderedItems` function, we might as well update this type too.